frontend/src/components/forms/CreateResident.tsx

```diff
@@ -132,7 +146,7 @@ const CreateResident = (): React.ReactElement => {
     }
 
     addResident();
-
+    setUserPageNum(1);
```

Comment:
I would just add this to `getResidents` where you called get and count above on line 74-75 to keep things consistent